src/main/scala/com/microsoft/hyperspace/actions/Action.scala

```diff
@@ -73,7 +73,7 @@ trait Action extends HyperspaceEventLogging with Logging with ActiveSparkSession
     }
   }
 
-  private def saveEntry(id: Int, entry: LogEntry): Unit = {
+  final protected def saveEntry(id: Int, entry: LogEntry): Unit = {
```

Comment:
Just a question: Why are these functions changed from private to protected?

Reply:
oh thanks for this. Looks like I missed some cleanup. This function doesn't need changing. I will revert.